gapic/src/main/com/google/gapid/views/StatusBar.java

```diff
@@ -101,6 +113,19 @@ public void setNotification(String text, Runnable onClick) {
     layout();
   }
 
+  public void setPerfettoStatus(String errorMsg) {
+    System.out.println("setPerfettoStatus");
+    System.out.println("errorMsg: " + errorMsg);
+    if (errorMsg == null) {
+      perfettoStatus.setVisible(false);
+    } else {
+      perfettoStatus.setVisible(true);
+      perfettoStatusText.setToolTipText(errorMsg);
```

Comment:
@pmuetschard @stellama0208 Could you help me out a bit here?
I am working on b/148797225 and the idea I have is to show a warning symbol in the status bar if there is any error while processing the trace.  When the user hover over the warning symbol, the tooltip will describe the problem.  However, setToolTipText doesn't seem to work.  Nothing shows up when hovering over the symbol.

Would you know why that is the case?  Or do you have alternative suggestions on how I should approach this?

Reply:
Is that because you are using `perfettoStatusText.setToolTipText(errorMsg);` rather than `perfettoStatus.setToolTipText(errorMsg);`, it seems that now the tooltips is on the text and not the symbol?